bin/.travis/test.sh

```diff
@@ -48,70 +51,73 @@ if [ "$REUSE_VOLUME" = "0" ]; then
         printf "\nAs requested will also force update packages after create-project\n"
         docker run -ti --rm \
           -e SYMFONY_ENV \
+          -e APP_ENV \
           -e PHP_INI_ENV_memory_limit=3G \
           -v $(pwd)/volumes/ezplatform:/var/www \
           -v  $COMPOSER_HOME:/root/.composer \
-          ez_php:latest-node \
+          ez_php:latest-dev \
           bash -c "
           composer --version &&
           composer create-project --prefer-dist --no-progress --no-interaction --no-scripts ezsystems/ezplatform /var/www $EZ_VERSION &&
           composer update --prefer-dist --no-progress --no-interaction --with-all-dependencies"
     else
         docker run -ti --rm \
           -e SYMFONY_ENV \
+          -e APP_ENV \
+          -e PHP_INI_ENV_memory_limit=3G \
```

Comment:
this is also temprary until we either don't pull in dev or until we start to pull in composer v2.